lisp_ease/src/queries/plans.lisp

```diff
@@ -0,0 +1,345 @@
+(in-package :le)
```

Comment:
Create a global parameter ``(defparameter *vr-inference-enabled* t)``. But not in this file please! Find a good place for it. Good place for this will be the user API place, where the user functions are defined, the topmost level. Per default it will have T. Define interface for Prolog predicates that will be used by VR and not only.
Prolog interface means something like below, whose implementation is ``(fail)``
```lisp
(<- (robot ?robot-name)
    (fail))
```